dotnet/src/SemanticKernel.Abstractions/Contents/ImageContent.cs

```diff
@@ -16,6 +16,11 @@ public sealed class ImageContent : KernelContent
     /// </summary>
     public Uri? Uri { get; set; }
 
+    /// <summary>
+    /// The Data used as DataUri for the image.
```

Comment:
@dersia Thanks for the changes.
I would remove the DataUri from the comment on this field and move it to the ToString.

```suggestion
    /// The image binary data.
```